scripts/ci/test_adequacy_lane.py

```
@@ -0,0 +1,347 @@
+        "survived": 4,
+        "score_percent": 60.0,
+        "control": control,
+        "tool_commit": commit,
+        "provenance": {"kind": kind},
+    }
+
+
+class CoverageAssertion(unittest.TestCase):
+    """Every way the lane could report a verdict it did not earn."""
+
+    PINS = {"a": TOOL_PIN, "b": TOOL_PIN}
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)